# -*- coding: utf-8 -*-
# Generated by Django 1.11.26 on 2019-11-17 21:03
from __future__ import unicode_literals

from django.db import migrations


class Migration(migrations.Migration):
    atomic = False

    dependencies = [
        ('fk', '0006_add_foreign_key_constraints'),
    ]

    operations = [
        # By calling RenameModel, for some reason Django no longer
        # fails to update the ForeignKey references in the schema.
        migrations.RenameModel('User', 'UserNew'),
        migrations.AlterModelTable(
            name='UserNew',
            table=None,
        ),
        migrations.RenameModel('UserNew', 'User'),
        migrations.AlterModelOptions(
            name='user',
            options={'verbose_name': 'user', 'verbose_name_plural': 'users'},
        ),
        migrations.AlterModelTable(
            name='category',
            table=None,
        ),
        migrations.RenameModel('FileFormat', 'FileFormatNew'),
        migrations.AlterModelTable(
            name='fileformatnew',
            table=None,
        ),
        migrations.RenameModel('FileFormatNew', 'FileFormat'),
        migrations.RenameModel('Organization', 'OrganizationNew'),
        migrations.AlterModelTable(
            name='organizationnew',
            table=None,
        ),
        migrations.RenameModel('OrganizationNew', 'Organization'),
        migrations.AlterModelTable(
            name='scheduleitem',
            table=None,
        ),
        migrations.RenameModel('Video', 'VideoNew'),
        migrations.AlterModelTable(
            name='videonew',
            table=None,
        ),
        migrations.RenameModel('VideoNew', 'Video'),
    ]
